snuba_sdk/query.py

```diff
@@ -51,6 +54,9 @@ class Query:
     offset: Optional[Offset] = None
     granularity: Optional[Granularity] = None
     totals: Optional[Totals] = None
+    consistent: Optional[Consistent] = None
+    turbo: Optional[Turbo] = None
+    debug: Optional[Debug] = None
```

Comment:
Since these are boolean flags they are either True or False. Do we need to make them optional ? That adds a third state (None) that we have to deal with every time we read the flag. IT seems they could have a default set to False. 
The flag object is immutable so you have no problems with mutable default (if they were mutable https://docs.python.org/3/library/dataclasses.html#default-factory-functions)

Reply:
I see. I should have looked at the JSON schema definition, I didn't realize they were being defaulted there. I'll default them to False in the SDK as well.